packages/app/src/game/module/board/overlays/readyOverlay.tsx

```diff
@@ -1,41 +1,62 @@
 import * as React from "react";
-import { useSelector } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
 import { GamePhase, PlayerListPlayer } from "@creature-chess/models";
 import { PlayerAvatar, Title, PlayerHealthbar } from "@creature-chess/ui";
 import { usePlayerId } from "../../../../auth";
 import { AppState } from "../../../../store";
 import { BoardOverlay } from "./boardOverlay";
 import { HeadToHeadStats } from "./h2h/headToHeadStats";
+import { QuickChatValue, QuickChatOption } from "@creature-chess/models";
+import { QuickChatBox } from "./quickChatBox";
+import { quickChatCommands } from "../../chat";
+import { QuickChatButton } from "./quickChatButton";
+import { PlayerActions, PlayerEvents } from "@creature-chess/gamemode";
 
 const ReadyOverlay: React.FunctionComponent = () => {
 
+	const dispatch = useDispatch();
+
 	const inReadyPhase = useSelector<AppState, boolean>(state =>
 		state.game.roundInfo.phase === GamePhase.READY
 	);
 
 	const playerList = useSelector((state: AppState) => state.game.playerList);
 
 	const localId = usePlayerId();
+	const sendingPlayerId = localId;
 	const localPlayer = playerList.find(p => p.id === localId);
 
 	const opponent: PlayerListPlayer = useSelector((state: AppState) => {
 		const id = state.game.playerInfo.opponentId;
 		return state.game.playerList.find(p => p.id === id);
 	});
+	const receivingPlayerId = opponent?.id;
 
 	const spectatingPlayer = useSelector<AppState, boolean>(state => state.game.spectating.id !== null);
 
 	if (!opponent || !inReadyPhase || spectatingPlayer) {
 		return null;
 	}
 
+	const sendQuickChat = (chatOption: QuickChatOption) => {
+		const chatValue: QuickChatValue = {
+			phrase: chatOption
```

Comment:
Just curious what made you choose to have this outer `QuickChatValue` object rather than just passing the `QuickChatOption` around directly (given that it only has 1 property)